src/test/shared/telemetry/defaultTelemetryService.test.ts

```diff
@@ -66,10 +68,14 @@ describe('DefaultTelemetryService', function () {
         service = initService()
         logger = service.logger
         clock = installFakeClock()
+        tempTelemetryFile = path.join(mockContext.globalStoragePath, 'TEMP_CACHE')
     })
 
     afterEach(async function () {
         await fs.remove(service.persistFilePath)
+        if (fs.existsSync(tempTelemetryFile)) {
+            await fs.remove(tempTelemetryFile)
```

Comment:
This remove() is from fs-extra, which does not throw if the file does  not exist. https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md